lib/blkdb.c

```diff
@@ -20,6 +20,13 @@
 #include <ccoin/cstr.h>
 #include <ccoin/compat.h>		/* for fdatasync */
 
+#if defined(__APPLE__)
+static int fdatasync(int fd)
+{
+	return fsync(fd);
+}
+#endif
+
```

Comment:
This is the only change that does not appear to make sense.

ccoin/compat.h already has code that provides a compatible fdatasync() for platforms that lack it.  Does that check/mechanism not work for you?